projects/RabbitMQ.Client/CreateChannelOptions.cs

```diff
@@ -46,6 +45,11 @@ public sealed class CreateChannelOptions
 
         /// <summary>
         /// Should this library track publisher confirmations for you? Defaults to <c>false</c>
+        ///
```

Comment:
How about adding a hint to PublisherConfirmationsEnabled that points to that header and indicates the header needs to be added manually when required by first acquiring a sequence number and point to the method?